packages/dev/core/src/Collisions/gpuPicker.ts

```diff
@@ -53,6 +69,28 @@ export class GPUPicker {
         return this._shaderLanguage;
     }
 
+    private static _IdToRgb(id: number) {

```

Comment:
Can we use a temp color3 or vector3 instead of creating a new object on every call ?

Reply:
Of course, thanks for the suggestion :-) Changed.